src/app/dynamic-forms/enums/data-format-type.enum.ts

```diff
@@ -0,0 +1,3 @@
+export enum DataFormatType {
+    IFrame = "iFrame"
```

Comment:
@cinchy-yogeshjagtap shouldn't we list all available Data Format Types (e.g., XML, image, etc.)?

Reply:
Hi @alvin-h , I have resolved above enum type issue by keeping only one enum ImageType. And club all above enums under it.